test/system/test_correct_genotypes_by_imputation.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+import subprocess
+from test import get_resource_dir, test_dir
+
+import pandas as pd
+import numpy as np
+
+from ugvc.pipelines import correct_genotypes_by_imputation as corgen
+from ugvc.vcfbed.vcftools import genotype_ordering
+
+class TestCorrectGenotypesByImputation:
+    inputs_dir = get_resource_dir(__file__)
+    resources_dir = get_resource_dir(__file__)
+
+    def test_convert_ds_to_genotype_imputation_priors(self):
```

Comment:
this is great.
It should be in test/unit/ though

Reply:
Also if you can separate each case to it's own test function that would be even better

Reply:
Problem with test/unit is that they do not refer to the files in the `pipelines` directory. Only system refers to pipeline.